apps/examples/testcase/le_tc/kernel/tc_pthread.c

```diff
@@ -1539,6 +1540,58 @@ static void tc_pthread_pthread_setcanceltype(void)
 }
 #endif
 
+/**
+* @fn                   :tc_pthread_pthread_mutex_consistent
+* @brief                :This tc tests pthread_mutex_consistent()
+* @Scenario             :the pthread_mutex_consistent() function can be used to mark the state protected by the mutex referenced by mutex as consistent again
+*                        and this situtaion arises, If an owner of a robust mutex terminates while holding the mutex,
+*                        the mutex becomes inconsistent and the next thread that acquires the mutex lock shall be notified of the state.
+*                        In this case, the mutex does not become normally usable again until the state is marked consistent.
+* @API'scovered         :pthread_setcanceltype
+* @Preconditions        :none
+* @Postconditions       :none
+* @return               :void
+*/
+static void *test_mutex_consistent(void *param)
+{
+	int ret = -1;
+	ret = pthread_mutex_lock(&g_mutex);
+	if (ret != 0) {
+		printf("pthread_mutexlock was not successful, errno = %d", ret);
+	}
+}
+
+static void tc_pthread_pthread_mutex_consistent(void)
+{
+	int ret;
+	pthread_t thread1;
+	pthread_t thread2;
+
+	pthread_mutexattr_init(&g_mutexattr);
+	pthread_mutexattr_setrobust(&g_mutexattr, PTHREAD_MUTEX_ROBUST);
+	pthread_mutex_init(&g_mutex, &g_mutexattr);
+
+	ret = pthread_create(&thread1, NULL, &test_mutex_consistent, NULL);
+	pthread_join(thread1, NULL);
+	TC_ASSERT_EQ("pthread_mutex_consistent", ret, 0);
+
+
+	ret = pthread_kill(thread1, 0);
```

Comment:
`pthread_kill` is for sending signal to the pthread.
So if you want to terminate the `thread1`, I think you should send signo `9`, instead of signo `0`.
How about using `pthread_cancel` if you want to terminate?

Reply:
will change in pthread_cancel. thanks

Reply:
Why? Could you let me know the reason? (I left the same comment above) Why do you need pthread_kill or pthread_cancel?

Reply:
@sunghan-chang I agree with your opinion. We don't need `pthread_cancel`, because `thread1` will be terminated automatically.

Reply:
yes sunghan, you are right. no need of pthread_cancel() or pthread_kill() because pthread_exit() is already called from the pthread function.